packages/manager/apps/pci-volume-backup/src/pages/create/steps/BackupOptionStep.tsx

```diff
@@ -0,0 +1,230 @@
+import { Trans, useTranslation } from 'react-i18next';
+
+import {
+  TAddon,
+  TileInputChoice,
+  TVolume,
+  useCatalog,
+  useProductAvailability,
+} from '@ovh-ux/manager-pci-common';
+import { OdsDivider, OdsLink, OdsText } from '@ovhcloud/ods-components/react';
+import { useContext, useEffect, useMemo } from 'react';
+import { useParams, useSearchParams } from 'react-router-dom';
+import { useMedia } from 'react-use';
+import {
+  ShellContext,
+  useOvhTracking,
+} from '@ovh-ux/manager-react-shell-client';
+import {
+  convertHourlyPriceToMonthly,
+  useCatalogPrice,
+} from '@ovh-ux/manager-react-components';
+import {
+  GUIDES,
+  GUIDES_STORAGES_VOLUME_BACKUP_OVERVIEW,
+  VOLUME_OPTION_BACKUP,
+  VOLUME_OPTION_SNAPSHOT,
+} from '@/constants';
+import { VOLUME_BACKUP_TRACKING } from '@/tracking.constant';
+
+export type TBackupOption = {
+  id: string;
+  planCode: string;
+  type: typeof VOLUME_OPTION_SNAPSHOT | typeof VOLUME_OPTION_BACKUP;
+  isDisabled?: boolean;
+  price: string | null;
+};
+
+type BackupOptionStepProps = {
+  selectedBackup: TBackupOption | undefined;
+  setSelectedBackup: (backup: TBackupOption) => void;
+  selectedVolume: TVolume | undefined;
+  onBackupChange: (option: TBackupOption) => void;
+  backupOptions: TBackupOption[];
+};
+
+export default function BackupOptionStep({
+  selectedBackup,
+  setSelectedBackup,
+  selectedVolume,
+  onBackupChange,
+  backupOptions,
+}: BackupOptionStepProps) {
+  const { t } = useTranslation(['create', 'pci-volume-backup']);
+  const { ovhSubsidiary } = useContext(ShellContext).environment.getUser();
+  const { trackClick } = useOvhTracking();
+
+  const isMobileView = useMedia(`(max-width: 36em)`);
+
+  const { projectId } = useParams();
+
+  const [searchParams] = useSearchParams();
+  const volumeOptionParam = searchParams.get('volumeOption');
+
+  const { data: catalog } = useCatalog();
+
+  const { getTextPrice } = useCatalogPrice(6);
+
+  const { data: snapshotAvailabilityData } = useProductAvailability(
+    projectId || '',
+    {
+      addonFamily: 'snapshot',
+    },
+  );
+
+  const { data: backupAvailabilityData } = useProductAvailability(
+    projectId || '',
+    {
+      addonFamily: 'volume-backup',
+    },
+  );
+
+  const backupGuideLink = useMemo(() => {
+    const guide = GUIDES.find(
+      ({ id }) => id === GUIDES_STORAGES_VOLUME_BACKUP_OVERVIEW,
+    );
+    return (
+      guide?.links[ovhSubsidiary as keyof typeof guide.links] ||
+      guide?.links.DEFAULT ||
+      ''
+    );
+  }, [ovhSubsidiary]);
```

Comment:
```suggestion
  const backupGuideLink = useMemo(() => {
    const guide = GUIDES.find(
      ({ id }) => id === GUIDES_STORAGES_VOLUME_BACKUP_OVERVIEW,
    );
    return (
      guide?.links[ovhSubsidiary as keyof typeof guide.links] ||
      guide?.links.EN ||
      ''
    );
  }, [ovhSubsidiary]);
```

The useMemo is unnecessary here. And Default can be directly replaced by EN for readability